relay-server/src/actors/processor.rs

```diff
@@ -370,6 +370,15 @@ fn outcome_from_parts(field: ClientReportField, reason: &str) -> Result<Outcome,
     }
 }
 
+pub fn outcome_from_profile_error(err: relay_profiling::ProfileError) -> Outcome {
```

Comment:
_nit_: This function looks like an implementation detail that doesn't have to be part of the public interface.

```suggestion
fn outcome_from_profile_error(err: relay_profiling::ProfileError) -> Outcome {
```